manual_test/manual_test_base.py

```diff
@@ -19,7 +19,7 @@ class ManualTestBase:
 
     disable_warnings(exceptions.InsecureRequestWarning)
 
-    def __init__(self, server: str, username: str, password: str, relax_validation: bool) -> None:
+    def __init__(self, server: str, relax_validation: bool, username: str = None, password: str = None, sle_token: str = None) -> None:
```

Comment:
Can we just call this an API Key? It's not a bearer token and we can use API keys with other products as well. No reason to be SLE-specific.